crates/hyperswitch_interfaces/src/consts.rs

```diff
@@ -11,3 +11,6 @@ pub const ACCEPT_HEADER: &str = "text/html,application/json";
 
 /// User agent for request send from backend server
 pub const USER_AGENT: &str = "Hyperswitch-Backend-Server";
+
+/// Unsupported response type error message
+pub const UNSUPPORTED_ERROR_MESSAGE: &str = "Unsupported response type";
```

Comment:
Can we move this to the crate where this is being used?

Reply:
Moved it to `crates/hyperswitch_connectors/src/constants.rs`